### ISSUE TITLE
What about using chips for labels in issues/PRs list?

### ISSUE THREAD

**Issue by Fs00** [slapperwan/gh4a#1219]
I think that [chips](https://material.io/components/chips) would be a nicer way to display labels in issues/PRs list instead of using simple colored dots as we are doing now.
It's often hard to remember which label has a particular color and also different labels can have similar colors (which are hard to distinguish since they are only dots).

**Comment by maniac103** [comment]
The reason labels are shown as dots in the list is me thinking adding chips with text will make the list to busy. There's repos that use 4 or 5 labels for each issue, and when spelling out the labels there, I think it's too distracting.

**Comment by Fs00** [comment]
Makes sense. One way to mitigate this could be to limit label chips to one line, and make the chip group horizontally scrollable when they don't fit in one line.
